packages/components/icons/generated/ImportReact/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "main": "dist/commercetools-uikit-icons-generated-ImportReact.cjs.js",
```

Comment:
Should we kebab case the package name part which is the icon?

Reply:
everything in `packages/icons/generated` is auto-generated by preconstruct when you run `preconstruct dev` or `preconstruct build`, and there is no way to configure this.